src/AppBundle/Helper/LabsHelper.php

```diff
@@ -111,6 +75,73 @@ public function databasePrepare($project = 'wiki')
     }
 
     /**
+     * Get the record for the given project in the meta.wiki table
+     * @param  string $project Valid project in the formats:
+     *                         https://en.wikipedia.org, en.wikipedia, enwiki
+     * @return string|false    database record or false if no record was found
```

Comment:
Does this really return a string? Regardless, either the function description or the param description should give a better description of what is actually returned, i.e. what is in this "database record"?